node/src/blockchain/blockchain_interface/rpc_helpers.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::blockchain::blockchain_interface::BlockchainResult;
+use crate::sub_lib::wallet::Wallet;
+use web3::types::U256;
+
+pub trait RPCHelpers {
```

Comment:
Let's find a common ground and migrate this code to a new card, because if this code says under this card, it'll cause conflicts with GH-744, and those conflicts won't be easy to resolve due to multiple calls to the `wait()` function.

I hope this comment aligns with your original idea to eliminate all the calls to the `wait()` function.

Reply:
This was a mistake. I got two versions of the same things during a merge. I deleted the outdated files.